source/geod24/LocalRest.d

```diff
@@ -103,6 +103,7 @@ module geod24.LocalRest;
 
 static import C = geod24.concurrency;
```

Comment:
our of spawn => out of spawn

Reply:
What ? Where is this written ?

Reply:
In the commit message. Sorry github makes it not easy to comment on the actual commit message.